docker-compose.yml

```diff
@@ -17,6 +17,7 @@ networks:
 services:
     mariadb:
         image: mariadb:latest
+        env_file: .env
```

Comment:
seems the `.env` file is missing from this commit.

Reply:
Sorry but for me the .env is never track by git